internal/provider/resource_pool.go

```diff
@@ -89,6 +98,10 @@ func resourcePoolRead(ctx context.Context, d *schema.ResourceData, m interface{}
 	d.Set("open_slots", pool.OpenSlots)
 	d.Set("used_slots", pool.UsedSlots)
 
+	// Manually manage include_deferred in Terraform state
+	includeDeferred := d.Get("include_deferred").(bool)
```

Comment:
Extraneous, let's align with other set examples on that are above this do d.Get(...) inline in the set call